application/Module.php

```diff
@@ -25,9 +25,6 @@ class Module extends AbstractModule
      */
     const OMEKA_VOCABULARY_TERM = 'o';
 
-    /**
-     * {@inheritDoc}
-     */
```

Comment:
I'm fine with removing these, I think...

This pattern originally came from following Zend Framework style guidance but they really are useless at best.

Reply:
Yes, the `@inheritDoc` should not be used alone, according to the phpdoc doc (https://docs.phpdoc.org/guides/inheritance.html), but some applications set it by default.